.github/dependabot.yml

```
@@ -23,3 +23,52 @@ updates:
     commit-message:
       prefix: "ci"
+
+# BEGIN: commit-message-standards
+# Dependabot config fragment. bootstrap-repo.sh patches these entries into
+# .github/dependabot.yml so Dependabot's commits pass the conventional-commits
+# check (Dependabot's default subject "Bump foo from 1 to 2" fails the format
+# rule per Eng E8). The `commit-message.prefix: "chore"` line is the
+# load-bearing part — everything else is sensible defaults.
+#
+# Spec reference: https://github.com/florianhorner/engineering-standards/blob/main/specs/commit-message-spec.md#bot-allowlist
+version: 2
```

YAMLlint (1.38.0)

[error] duplication of key "version" in mapping

(key-duplicates)